sklearn/preprocessing/tests/test_polynomial.py

```diff
@@ -75,7 +92,8 @@ def test_spline_transformer_input_validation(params, err_msg):
 
 
 def test_spline_transformer_manual_knot_input():
-    """Test that array-like knot positions in SplineTransformer are accepted.
+    """
+    Test that array-like knot positions in SplineTransformer are accepted.
```

Comment:
Why?

Reply:
Flake8 was complaining. See https://github.com/scikit-learn/scikit-learn/pull/19483/commits/a6e79e2c9a1e9ae597a01e7163e044174576451c and https://github.com/scikit-learn/scikit-learn/pull/19483/commits/8d8a2195b09f1d2efeebe77178a32a1f51c1221a.